core/chains/evm/txm/dummy_keystore.go

```diff
@@ -8,15 +8,17 @@ import (
 
 	"github.com/ethereum/go-ethereum/accounts"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/crypto"
 )
 
 type DummyKeystore struct {
+	chainID       *big.Int
 	privateKeyMap map[common.Address]*ecdsa.PrivateKey
 }
 
-func NewKeystore() *DummyKeystore {
-	return &DummyKeystore{privateKeyMap: make(map[common.Address]*ecdsa.PrivateKey)}
+func NewKeystore(chainID *big.Int) *DummyKeystore {
```

Comment:
What is using this?

Reply:
This is a basic keystore implementation that can be used if someone wants to run TXMv2 outside of the Core node.